.github/workflows/functional_all_db.yml

```diff
@@ -162,7 +162,10 @@ jobs:
       OrchardCore__DatabaseProvider: "SqlConnection"
     steps:
     - uses: actions/checkout@v3
-    - uses: actions/setup-dotnet@v3
+    - name: Allows NodeJs to find the dotnet command
+      run: |
+      shell: pwsh
+          $Env:Path += [IO.Path]::PathSeparator + "/usr/share/dotnet"
```

Comment:
I tried something similar but it didn't work for me, if that's working then it's great. That's weird though that it broke at some point, maybe they have an issue with their base images.

Reply:
Could it be part of the js script? Or could the js script try to perfix dotnet with this path (which is what I had tried I think).

Reply:
Yes, will see

Now it fails because it doesn't find powershell ;)

    OCI runtime exec failed: exec failed: unable to start container process: exec: "pwsh": executable file not found in $PATH: unknown

Reply:
> Could it be part of the js script?

I will check

Reply:
Failing only with SQLServer, not sure why

Reply:
Because for testing I only updated the SQLServer test

So it means that we would also need to update the PATH for powershell or use directly the  available command line in the containers we have initialized.